source/release-notes.txt

```diff
@@ -10,6 +10,48 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.20
+--------------
+
+*Released November, 2019*
+
+- New and improved :guilabel:`New Connection` user experience with
+  support for all connection options.
+
+- Added support for:
+
+  - Cloning :guilabel:`Favorites`.
+
+  - Adding, viewing and editing documents in JSON.
+
+  - Querying UUIDs via the :guilabel:`Documents` query bar or in
```

Comment:
[nit] I would replace "Aggregations" with "the Aggregation Pipeline Builder". Also, rather than :guilabels: I would link to the respective Documents and Agg Pipeline documentation pages.